launcher/cog-launcher.c

```diff
@@ -1028,6 +1047,32 @@ option_entry_parse_gamepad(const char *option_name, const char *value, void *dat
     return TRUE;
 }
 
+#if HAVE_WEBKIT_AUTOPLAY
+static gboolean
+option_entry_parse_autoplay(const char *option_name, const char *value, void *data, GError **error)
+{
+    if (!g_strcmp0(value, "allow")) {
```

Comment:
We could match the passed value against the possible values for the GEnum class by using `g_enum_get_value_by_nick()` instead of manually checking possible values. I think it's fine to merge this patch as-is and do that as a follow-up later on.